mongodb-panache-quickstart/src/main/java/org/acme/mongodb/panache/Person.java

```diff
@@ -5,11 +5,14 @@
 
 import org.bson.codecs.pojo.annotations.BsonProperty;
 
+import com.fasterxml.jackson.annotation.JsonProperty;
+
 import io.quarkus.mongodb.panache.MongoEntity;
 import io.quarkus.mongodb.panache.PanacheMongoEntity;
 
 @MongoEntity(collection = "ThePerson")
 public class Person extends PanacheMongoEntity {
+    @JsonProperty
```

Comment:
@loicmathieu I'm not really happy about this change but, when switching to Jackson, I end up with `name` being null in the deserialized `Person`.

Reply:
It's strange.
I remember noticing that Jackson and JSON-B are not configured the same way. And Restassured used a different ObjectMapper configuration than the application.

Anyway, your change is strange, it should not have any impact and `name` should not be null.
If you want to merge your PR with this change, create a followup issue so someone (me or @geoand, the master of serialization ;) ) can have a look.

Reply:
Yeah, I had a PR to make RestAssured use the applications's `ObjectMapper`, but I never merged it because it does not work in native tests and would therefore create far too much confusion.

Reply:
so `name` is serialized OK by Quarkus but somehow the ObjectMapper of REST Assured doesn't deserialize it properly. What is extremely weird is that REST Assured was already using Jackson.